### DISCUSSION TITLE
GHC 9.2.7

### DISCUSSION THREAD

**Discussion by StevenXL** [LazyVim/LazyVim#2443] [Q&A]
Hey folks. 

Whenever I open a Haskell file, I get the following error:

![Screenshot from 2024-01-27 15-14-12](https://github.com/LazyVim/LazyVim/assets/7391433/af3aab84-4077-4e3d-b16f-6261f0baccbc)

What's throwing this error, and why can't it find this https://github.com/haskell/haskell-language-server/releases/tag/1.10.0.0 which supports 9.2.7? 

I tried viewing MasonLogs but didn't see anything there.

**Comment by brgr-s** [comment]
@StevenXL did you ever manage to fix this? running into the same problem.
